src/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+add_subdirectory(Interface)
+add_subdirectory(Optimization)
+add_subdirectory(LinAlg)
+add_subdirectory(Drivers)
```

Comment:
I am not sure if this matters, but I would add `Drivers` directory at the very end of this file after HiOp libraries are built. If nothing else, it makes the logic of the CMake code easier to follow.

Reply:
Great point!